packages/jest-preset/transform.js

```diff
@@ -2,21 +2,6 @@ var babelJest = require('babel-jest');
 
 var hopsBabelConfig = require('hops-config/loaders/babel').node.use.options;
 
-// presets look like this
-// we try to figure out at which position "env" is and and set modules to "commonjs" to transpile import statements in jest
```

Comment:
I agree with removing the comment about the structure, however personally I'd leave the apart about why this is happening here (allowing import statements in jest)